bootstrap/sql/migrations/native/1.4.0/mysql/postDataMigrationSQLScript.sql

```diff
@@ -363,22 +363,26 @@ set json = JSON_INSERT(
 WHERE name = 'tableRowInsertedCountToBeBetween';
 -- End of Test Definition Parameter Definition Validation Migration
 
--- update existing customLogoConfiguration config with new customUiThemePreference
-UPDATE openmetadata_settings
+-- Start of updating existing customLogoConfiguration config with new customUiThemePreference
+UPDATE openmetadata_settings AS ui
+JOIN openmetadata_settings AS logo
```

Comment:
Maybe I am missing something but wondering why we do that inner join

Reply:
the self join is required as we need to  to move the old config values stored in "customLogoConfigurationType" to new "customUiThemePreference" configType